packages/lodestar-cli/src/wallet/Wallet.ts

```diff
@@ -43,7 +43,7 @@ export class Wallet extends Keystore {
    */
   static async fromMnemonic(mnemonic: string, password: string, name: string): Promise<Wallet> {
     const seed = deriveKeyFromMnemonic(mnemonic);
-    const publicKey = generatePublicKey(seed);
+    const publicKey = bls.PrivateKey.fromBytes(seed).toPublicKey().toBytes();
```

Comment:
All this code should be removed, we should not handle validator creation in the CLI

Reply:
This is just to fix build errors. We can tackle this as a separate PR